OpenWeb-SampleApp/UI/AuthenticationPlayground/ViewModels/AuthenticationPlaygroundViewModel.swift

```diff
@@ -225,13 +234,24 @@ private extension AuthenticationPlaygroundViewModel {
                 self?._thirdPartySSOAuthenticationStatus.onNext(.initial)
                 self?._logoutAuthenticationStatus.onNext(.initial)
             })
-            .withLatestFrom(shouldInitializeSDK) { genericSSO, shouldInitializeSDK -> GenericSSOAuthentication in
+            .withLatestFrom(shouldInitializeSDK) { ($0, $1) }
+            .withLatestFrom(customAuthOn) { ($0.0, $0.1, $1) }
+            .withLatestFrom(customUsername) { ($0.0, $0.1, $0.2, $1) }
+            .withLatestFrom(customPassword) { ($0.0, $0.1, $0.2, $0.3, $1) }
+            .withLatestFrom(customSSOToken) { ($0.0, $0.1, $0.2, $0.3, $0.4, $1) }
```

Comment:
Wow this is confusing. Doesn't `combineLatest` do that?

Reply:
But if I will put this in a combine latest then they all will be a trigger, I don't want every custom field/switch to trigger this authentication

Reply:
This is also a sample app authentication code, and I think this is ok :)
What do you think?

Reply:
I will try

Reply:
Found a little cleaner way to do this :)
```swift
.withLatestFrom(
                Observable.combineLatest(shouldInitializeSDK,
                                         customAuthOn,
                                         customUsername,
                                         customPassword,
                                         customSSOToken)
            ) { genericSSO, latestValues in
                return (genericSSO, latestValues.0, latestValues.1, latestValues.2, latestValues.3, latestValues.4)
            }
            .flatMapLatest { genericSSO, shouldInitializeSDK, customAuthOn, customUsername, customPassword, customSSOToken -> Observable<GenericSSOAuthentication> in
``` 

I have tested and pushed this change.